index.js

```diff
@@ -2,8 +2,11 @@
 var objectAssign = require('object-assign');
```

Comment:
Not sure, I'll let @sindresorhus decide. This PR already drops 0.10 and 0.12 which might be to much for only adding submenus. So either we use a temporary ponyfill like [repeating](https://github.com/sindresorhus/repeating) and do an ES2015ify + cleanup afterwards.